bearer/bearer.go

```diff
@@ -262,6 +262,15 @@ func (b Token) AssertUser(id user.ID) bool {
 	return !b.targetUserSet || b.targetUser.Equals(id)
 }
 
+// SignByIssuer combines [Token.Sign] and [Token.SetIssuer] in one call. Use
+// this method for stable authorization in the system. SignByUser should not be
+// mixed with the mentioned methods.
+func (b *Token) SignByIssuer(signer user.Signer) error {
```

Comment:
you do this one way and the next commit does it another way, why? why not just `Sign` that sets issuer?

Reply:
im constrained by what we already have:
* bearer `Sign` accepts generic signer w/o an issuer, so new method is needed
* session `Sign` does, so new method is needed for generic signature setting

sync would be breaking overall, but i agree that unified interface is easier to follow

@roman-khimov @smallhive

Reply:
I'd vote for `Sign(signer user.Signer)` unification, but I fear @amlwwalker will hate us for changing the interface again.

Reply:
I don't hate anything you guys are doing @roman-khimov just have to keep up! 😂